torch/_dynamo/config_utils.py

```diff
@@ -0,0 +1,44 @@
+import inspect
+import pickle
+
+
+# Construct functions that save/load the state of the config module `module`.
+# The config settings are expected to either be module-level globals or
+# class variables.
+# `ignore_set` is a set of names of configurations to ignore. e.g. if you
+# want to ignore config.x and config.y.z in your config module, then
+# `ignore_set` should be {"x", "y.z"}.
+def get_config_serialization_fns(module, ignore_set=None):
```

Comment:
Shouldn't we just override `__getstate__` and `__setstate__` for the `_AccessLimitingConfig`

Reply:
Inductor config doesn't use a class such as _AccessLimitingConfig.